src/libraries/Common/src/Interop/SunOS/procfs/Interop.ProcFsStat.cs

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class @procfs
```

Comment:
Is it missing the functionality or here to-be-filled in the future? In case it's the latter, lets delete it for now.

Reply:
It looks like the original idea was to split up that code.  That may still make sense later -- I'm not sure.
I find this file name: Interop.ProcFsStat.TryReadProcessStatusInfo.cs to be annoyingly long.
How about I delete that and put this all in Interop.ProcFsStat.cs instead?

Reply:
Based on https://github.com/dotnet/runtime/blob/dff1d8467845fd93c517f89dc81598e5fd17c270/docs/coding-guidelines/interop-guidelines.md, we can move:

* struct definitions in `Interop.ProcFs.Definitions.cs`
* then for each internal method like TryReadProcessStatusInfo, TryReadProcessThreadInfo etc. create a separate Interop.ProcFs.XX.cs file.

Linux procfs is not using this approach because it wasn't upgraded when guidelines were written, but for new platform port, we can follow the guidelines to keep things tidy.

If something (struct, const or a method method) is not used outside the `@procfs` class, change `internal` to `private`.

Reply:
> Based on https://github.com/dotnet/runtime/blob/dff1d8467845fd93c517f89dc81598e5fd17c270/docs/coding-guidelines/interop-guidelines.md, we can move:
> 
> * struct definitions in `Interop.ProcFs.Definitions.cs`
> * then for each internal method like TryReadProcessStatusInfo, TryReadProcessThreadInfo etc. create a separate Interop.ProcFs.XX.cs file.
> 

I tried moving some of the structs and stuff like RootPath but then the C# compiler did not "know about" the structures in the other file.  Any tips on how to do that?  Thanks!

Reply:
Did you add lines for new files in src/libraries/System.Diagnostics.Process/src/System.Diagnostics.Process.csproj?

Reply:
Yes, and it seems like that file is being ignored.  New files I add are apparently ignored. Files I've removed are still being looked for, and I don't see why.  Eg.
```
CSC : error CS2001: Source file '/home/gwr/dotnet/runtime/src/libraries/Common/src/Interop/SunOS/procfs/Interop.ProcFsStat.TryReadProcessStatusInfo.cs' could not be found. [/home/gwr/dotnet/runtime/src/coreclr/System.Private.CoreLib/System.Private.CoreLib.csproj]
```
What am I missing?

Reply:
Oh, found it:
```
src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems
```